src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/DeveloperExperience/DeveloperExperience.cs

```diff
@@ -65,7 +66,7 @@ public virtual string CreateStackTraceString(IntPtr ip, bool includeFileInfo)
             }
 
             StringBuilder sb = new StringBuilder();
-            string fileNameWithoutExtension = GetFileNameWithoutExtension(moduleFullFileName);
+            string fileNameWithoutExtension = Path.GetFileNameWithoutExtension(moduleFullFileName);
             int rva = (int)(ip.ToInt64() - moduleBase.ToInt64());
```

Comment:
```suggestion
            ReadOnlySpan<char> fileNameWithoutExtension = Path.GetFileNameWithoutExtension(moduleFullFileName.AsSpan());
            int rva = (int)(ip - moduleBase);
```
Avoid allocation and simplify